.travis.yml

```diff
@@ -44,6 +44,16 @@ script:
 
 jobs:
   include:
+    # only test build using meson on freebsd, the rest is covered by linux
+    - stage: test
+      os: freebsd
+      compiler: clang
+      install:
+        - sudo pkg install -y meson
```

Comment:
we install from pip for ubuntu (because it's old), for consistency we should do the same here I think

Reply:
FreeBSD doesn't support old packages: `/quarterly` (default) has meson 0.53.2, `/latest` has 0.54.2. Installing from pip means trusting upstream projects to QA on FreeBSD but Meson project doesn't seem to.

Reply:
ack

Reply:
Language-specific package managers tend to have trouble with foreign build systems on non-mainstream platforms e.g.,
```
$ sudo pkg install -qy py37-pip
$ pip install meson ninja
Collecting meson
  Downloading https://files.pythonhosted.org/packages/78/14/53d77eeb196efb1230eedaba556c920fc59989f1ebd429e990cb129b41d3/meson-0.54.2.tar.gz (1.7MB)
     |████████████████████████████████| 1.7MB 724kB/s
  Installing build dependencies ... done
  Getting requirements to build wheel ... done
    Preparing wheel metadata ... done
Collecting ninja
  Downloading https://files.pythonhosted.org/packages/dd/96/e2ec4acccb8dee33b4987f553d531d61e3081c8d4cfbce249655dfe23906/ninja-1.9.0.post1.tar.gz
    ERROR: Complete output from command python setup.py egg_info:
    ERROR: Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "/tmp/pip-install-ef8x46bl/ninja/setup.py", line 7, in <module>
        from skbuild import setup
    ModuleNotFoundError: No module named 'skbuild'
    ----------------------------------------
ERROR: Command "python setup.py egg_info" failed with error code 1 in /tmp/pip-install-ef8x46bl/ninja/
```